firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift

```diff
@@ -222,6 +222,7 @@ final class ToolbarMiddleware: FeatureFlaggable {
     }
 
     // MARK: - Border
+    // For the top placement of the address bar, the border is only visible on scroll
```

Comment:
do you know the decision behind this, would it be helpful to add a comment on why we are doing this? For user experience?

Reply:
In Figma it's written that it's a design choice to only show the border on scroll for the top address toolbar placement.

Reply:
thank you!